File: aspire-deployment/before/ShoeMoney.Common/Data/Entities/OrderItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ShoeMoney.Data.Entities;

public class OrderItem
{
  public int Id { get; set; }
  public int ProductId { get; set; }
  public Product? Product { get; set; }
  public string? Size { get; set; }
  public string? Width { get; set; }
  public decimal Price { get; set; }
  public decimal Quantity { get; set; }
  public float Discount { get; set; }
  public int OrderId { get; set; }
}

File: aspire-addjs/ShoeMoney.Store/src/models/OrderItem.ts
import {type Product} from "./Product";

export interface OrderItem {
  id: number;
  productId: number;
  product: Product | null;
  price: number;
  quantity: number;
  discount: number;
  orderId: number;
  size: string | null;
  width: string | null;
}